pkg/nuclide-file-tree/components/FileTreeEntryComponent.js

```diff
@@ -284,7 +284,7 @@ export class FileTreeEntryComponent extends React.Component {
     const nativeEvent = (event.nativeEvent: any);
     nativeEvent.dataTransfer.effectAllowed = 'move';
     nativeEvent.dataTransfer.setDragImage(fileIcon, -8, -4);
-
+    nativeEvent.dataTransfer.setData("initialPath", this.props.node.uri);
```

Comment:
For future contributions, you can use the `linter-eslint` Atom package to warn you of issues from within Atom. You can also run `npm run lint-all --ignore-scripts false` from the root of the `nuclide` repository to check for lint errors (this should probably be in CONTRIBUTING.md or something).

Our CI prevents changes with lint errors from getting committed so things need to be lint-clean to get accepted.